test/machineslib.py

```diff
@@ -19,29 +19,34 @@
 import sys
 import time
 import traceback
+import unittest
+from collections.abc import Mapping, Sequence
 
 import netlib
 import storagelib
 import testlib
 from machine import testvm
 
 
-def hasMonolithicDaemon(image):
+def hasMonolithicDaemon(image: str) -> bool:
     return (image.startswith("rhel-8-") or
             image.startswith("debian") or
             image.startswith("ubuntu") or
             image in ["arch"])
 
 
 # software TPM doesn't work on some OSes
-def hasBrokenTPM(image):
+def hasBrokenTPM(image: str) -> bool:
     return image.startswith("rhel-8")
 
 
-class VirtualMachinesCaseHelpers:
+class VirtualMachinesCaseHelpers(unittest.TestCase):
```

Comment:
This is wrong in two different ways:
 * Deriving from unittest.TestCase doesn't give you the `allow_journal_messages()` declaration.
 * This class was explicitly written to be a mix-in. Nothing else apart from `VirtualMachinesCase` uses it any more though

I rather suggest to just move everything into `VirtualMachinesCase` (which already derives from our `MachinesCase`).

Reply:
Just rebase on https://github.com/cockpit-project/cockpit-machines/pull/2059 and we should be good

Reply:
That PR landed, unblocked